OJP/OJP_Trips.xsd

```diff
@@ -1380,24 +1380,21 @@ Can only be used when both participants recognise the same schedule version. If
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="BaseTripPolicyGroup"/>
-			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" default="anyPoint">
+			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" minOccurs="0" default="anyPoint">
 				<xs:annotation>
-					<xs:documentation>If a solution for any one of multiple origin/destination points is sufficient. Or a distinct solution for each of the origin/destination points has to be found.</xs:documentation>
+					<xs:documentation>Defines how the router should handle requests with multiple origins and destinations. As it is important for the strategy of the distributed trip planning the MultiPointType should be set. If the type is not supported a TRIP_MULTIPOINT_TYPE_NOT_SUPPORTED warning or error must be returned. Default is anyPoint.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
 		</xs:sequence>
 	</xs:group>
 	<xs:simpleType name="MultiPointTypeEnumeration">
```

Comment:
What is the difference of `anyPoint` vs `somePoints`?

If I understand it correctly `anyPoint` should return only one trip result, while `somePoints` should return multiple based on some ("probably the best" ?!) selected origin/destination pairs.

But the description of `MultiPointTypeEnumeration` says "`MultiPointType` is more important than `NumberOfResults`. If the algorithm deems it necessary to return more results, it should do so."

So returning multiple results on `anyPoint` is considered valid anyway?

Reply:
In anyPoint then it is results from one origin and one destination, but it can still be 10 results all coming from the same pair.
In somePoint one does not know what number of different origins is used and also destinations. It is on up to n.

And yes, having anyPoint with 10 results is valid.